text/0000-wallet-replay-protection.md

```diff
@@ -0,0 +1,298 @@
+
+- Title: `wallet-(re)play-protection`
+- Authors: Phyro, John Tromp, Yeastplume
+- Start date: Aug 9, 2020
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000) 
+- Tracking issue: [Edit if merged with link to tracking github issue]
+
+---
+
+# Summary
+[summary]: #summary
+
+A few months ago, a new class of attacks was found on Grin that we call (re)play attacks. A Mimblewimble transaction that already happened can be replayed if the exact conditions are recreated. In order to be able to replay a transaction the following conditions must be true:

```

Comment:
I'd avoid referencing time like "a few months ago". Instead write the RFC like it's going to be there for years to come and that it should always read as up to date.